bosk-mongo/src/main/java/io/vena/bosk/drivers/mongo/v2/MainDriver.java

```diff
@@ -152,7 +163,22 @@ private void recoverFrom(Exception exception) {
 		}
 	}
 
+	private void backgroundReconnectTask() {
+		boolean isDisconnected = formatDriver instanceof DisconnectedDriver;
+		if (isDisconnected) {
```

Comment:
I wanted to make this as cheap as possible when there's nothing to do. So I don't even set up the `MDCScope` if the driver is not disconnected.

Reply:
Makes sense